prometheus/Dockerfile.prometheus

```diff
@@ -1,5 +1,5 @@
-FROM prom/prometheus:v1.5.2
+FROM prom/prometheus:v2.0.0
```

Comment:
At this point we can delete the custom images by mounting the configs instead of writing them into custom Docker images.

Reply:
Agreed. I'll update the compose file to just use the `prom/` image and mount the two config files.

On a related note, we removed alert.rules from the repo 4 days ago. 
https://github.com/openfaas/faas/commit/01450bdf17c60a20bfa79e59b52df3c5ddd05a2d#diff-e4f00638b8a10e6994e67af2f832d51c

As long as we're using `v1.x` isnt that file needed? (If not, the `k8s.alert.rules` file should also be superfluous?)